roles/ceph-mon/tasks/docker/fetch_configs.yml

```diff
@@ -21,7 +21,7 @@
 
 - name: try to fetch ceph config and keys
   copy:
-    src: "{{ playbook_dir }}/{{ fetch_directory }}/docker_mon_files/{{ item.0 }}"
+    src: "{{ fetch_directory }}/docker_mon_files/{{ item.0 }}"
```

Comment:
That's the current guidance, but only because of https://github.com/ansible/ansible/issues/18090. Once that bug is fixed we should be able to run playbooks from within `infrastructure-playbooks`.